src/error.rs

```diff
@@ -0,0 +1,43 @@
+use platform_impl;
+
+pub enum ExternalError {
+    NotSupported(NotSupportedError),
+    Os(OsError),
+}
+
+/// TODO: MANUALLY IMPLEMENT TRAITS
```

Comment:
But why?

Reply:
That was mainly referring to `Debug` impls, since otherwise the `_marker` field will show up in the auto-generated impl.